src/engine/EventManager.hxx

```diff
@@ -27,6 +27,8 @@ private:
 
   UIElement *m_lastHoveredElement = nullptr;
 
+  /// remember if placement is allowed from mousemove to mousedown
+  bool placementAllowed = false;
```

Comment:
I think we should reserve syntax.
we should use m_placementAllowed (m indicates that this is a member of a class)
it will be more readable inside code.

Reply:
added the m_